lib/shared/addon/components/aks-node-pool-row/component.js

```diff
@@ -27,9 +27,62 @@ export default Component.extend({
   upgradeVersion:                  false,
   showNodeUpgradePreventionReason: false,
   disableAzs:                      false,
+  hasLabels:                       false,
+  taints:                          {},
+  hasTaints:                       false,
 
   isSystemType: equal('nodePool.mode', 'System'),
 
+  init() {
+    this._super(...arguments);
+
+    const taints = get(this, 'nodePool.nodeTaints');
+
+    if (taints) {
+      let _taints = taints.map((t = '') => {
+        const splitEffect = t.split(':');
+        const splitLabel = (splitEffect[0] || '').split('=');
+
+        return {
+          effect: splitEffect[1],
+          key:    splitLabel[0],
+          value:  splitLabel[1],
+        };
+      })
+
+      set(this, 'taints.nodeTaints', _taints);
```

Comment:
I think I found a solution for the taint bug we saw [here](https://github.com/emberjs/data/issues/3276#issuecomment-110435315)
... it looks like when `nodeTaints` is set this way, the aks-node-pool-row component prototype is mutated, so each instance of aks-node-pool-row is essentially sharing the same array of taints. Going off the solution I linked above, you can fix the taint bug by deleting the `taints` definition on line 31 doing something like 
```
this.taints = {
   nodeTaints: _taints
}
```

and likewise replacing line 56 with 
```
this.taints = {
   nodeTaints: []
}
```

I tried this out myself and creation/editing all seemed fine with various configurations of multiple node pools with and without taints - I'd push the change up myself but don't have permission to do so.

Reply:
Thanks @mantis-toboggan-md - that worked. All updated now and should be ready for review.